src/scripts/main.js

```diff
@@ -1,3 +1,390 @@
 'use strict';
 
-// write your code here
+const start = document.querySelector('.start');
+const score = document.querySelector('.game-score');
+const messageStart = document.querySelector('.message-start');
+const messageLose = document.querySelector('.message-lose');
+const messageWin = document.querySelector('.message-win');
+
+score.textContent = 0;
+
+const cells = {
+  1: '',
+  2: '',
+  3: '',
+  4: '',
+  5: '',
+  6: '',
+  7: '',
+  8: '',
+  9: '',
+  10: '',
+  11: '',
+  12: '',
+  13: '',
+  14: '',
+  15: '',
+  16: '',
+};
+
+const colorMods = [
+  'field-cell--2',
+  'field-cell--4',
+  'field-cell--8',
+  'field-cell--16',
+  'field-cell--32',
+  'field-cell--64',
+  'field-cell--128',
+  'field-cell--256',
+  'field-cell--512',
+  'field-cell--1024',
+  'field-cell--2048',
+];
+
+function getRandom(min, max) {
+  return Math.round((min - 0.5) + Math.random() * (max - min + 1));
+}
+
+function removeColors() {
+  for (const key in cells) {
+    for (const mod of colorMods) {
+      if (document.querySelector(`.cell_${key}`).classList.contains(mod)) {
+        document.querySelector(`.cell_${key}`).classList.remove(mod);
+      }
+    }
+  }
+}
+
+function sync() {
+  for (const key in cells) {
+    document.querySelector(`.cell_${key}`).textContent = cells[key];
+
+    switch (cells[key]) {
```

Comment:
what about smth like

```
if (cells[key]) {
  document.querySelector(`.cell_${key}`).classList.add(`field-cell--${cells[key]}`)
}
```
instead of the entire switch block (you can save cells[key] in variable)